ToyBox/classes/MainUI/PartyEditor/StatsEditor.cs

```diff
@@ -88,6 +88,19 @@ public partial class PartyEditor {
                 Space(528);
                 ActionButton("Reset", () => { ch.Descriptor.State.Size = ch.Descriptor.OriginalSize; }, Width(197));
             }
+            using (HorizontalScope()) {
+                if (ch != null && ch.HashKey() != null) {
+                    Space(100);
+                    var scaleMult = Main.settings.perSave.scaleVisualModelSettings.GetValueOrDefault(ch.HashKey(), 1.0f);
```

Comment:
This will get invoked 60 times a second.  Please consider reading the value once and saving it into a member.